docs/books/admin_guide/13-softwares.md

```diff
@@ -523,6 +523,168 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-rockyofficial # GPG public key path
+
+```
+dnf module list
+```
+
+This will give you a long list of the available modules and the profiles that can be used for them. The thing is you probably already know what package you are interested in, so to find out if there are modules for a particular package, add the package name after "list". We will use our `postgresql` package example again here:
+
+```
+dnf module list postgresql
+```
+
+This will give you output that looks like this:
+
+```
+Rocky Linux 8 - AppStream
+Name                       Stream                 Profiles                           Summary                                            
+postgresql                 9.6                    client, server [d]                 PostgreSQL server and client module                
+postgresql                 10 [d]                 client, server [d]                 PostgreSQL server and client module                
+postgresql                 12                     client, server [d]                 PostgreSQL server and client module                
+postgresql                 13                     client, server [d]                 PostgreSQL server and client module
+```
+
+Notice in the listing the "[d]". This means that this is the default. It shows that the default version is 10 and that regardless of which version you choose, if you do not specify a profile, then the server profile will be the profile used, as it is the default as well.
+
+### Enabling Modules
+
+Using our example `postgresql` package, let's say that we want to enable version 12. To do this, you simply use the following:
+
+```
+dnf module enable postgresql:12
+```
+
+Here the enable command requires the module name followed by a ":" and the stream name.
+
+To verify that you have enabled `postgresql` module stream version 12, use your list command again which should show you the following output:
+
+```
+Rocky Linux 8 - AppStream
+Name                       Stream                 Profiles                           Summary                                            
+postgresql                 9.6                    client, server [d]                 PostgreSQL server and client module                
+postgresql                 10 [d]                 client, server [d]                 PostgreSQL server and client module                
+postgresql                 12 [e]                 client, server [d]                 PostgreSQL server and client module                
+postgresql                 13                     client, server [d]                 PostgreSQL server and client module
+```
+
+Here we can see the "[e]" for "enabled" next to stream 12, so we know that version 12 is enabled.
+
+### Installing Packages From The Module Stream
+
+Now that our module stream is enabled, the next step is to install `postgresql`. In this example, let's assume that we only want the client profile applied to our installation. To do this, we simply enter this command:
```

Comment:
This reads like it's only possible to install packages through profiles, might need a rewording to tell that you can install the wanted packages/package versions after enabling the stream, OR you can basically skip both enabling and manually installing and just do a `dnf module install postgresql:13/client`, which does install the packages in the profile + enables the stream!